test/FunctionalTests/Diagnostics/DiagnosticsStatusCodesMvcTests.cs

```diff
@@ -38,6 +38,7 @@ public class DiagnosticsStatusCodesMvcTests
         [OSSkipCondition(OperatingSystems.Windows)]
         [InlineData(ServerType.Kestrel, RuntimeFlavor.Clr, RuntimeArchitecture.x64, "http://localhost:6109")]
         [InlineData(ServerType.Kestrel, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, "http://localhost:6110")]
+        [InlineData(ServerType.Nginx, RuntimeFlavor.CoreClr, RuntimeArchitecture.x64, "http://localhost:6111")]
```

Comment:
Fix the port conflicts between this test and the other ones. Might want to check you other changes too.